### PR TITLE
Fix dependencies to correct version and ids

### PR DESCRIPTION
**Pull request by aefimov**

### CODE REVIEW

File: cloud-openstack-agent/pom.xml [hunk 1]
```diff
@@ -21,16 +21,15 @@
         <dependency>
             <groupId>org.jetbrains.teamcity</groupId>
             <artifactId>agent-api</artifactId>
+            <version>${teamcity.version}</version>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
-            <groupId>jetbrains.buildServer.agent</groupId>
+            <groupId>org.jetbrains.teamcity.internal</groupId>
```
Comment (mavlyutov) [inline cloud-openstack-agent/pom.xml]: is this change belongs to 8.0 -> 9.0 migration?

Comment (aefimov) [inline cloud-openstack-agent/pom.xml]: http://repository.jetbrains.com/all/org/jetbrains/teamcity/internal/agent/

File: pom.xml [hunk 2]
```diff
@@ -34,8 +34,8 @@
         <dependencies>
             <!-- these are opensourced teamcity dependencies-->
             <dependency>
-                <groupId>com.intellij</groupId>
-                <artifactId>openapi</artifactId>
+                <groupId>org.jetbrains.teamcity.idea</groupId>
+                <artifactId>obsolete-openapi</artifactId>
```
Comment (mavlyutov) [inline pom.xml]: why do you plan to use obsolete one? is there any *new* version of openapi?

File: pom.xml [hunk 3]
```diff
@@ -66,12 +67,6 @@
                 <version>${teamcity.version}</version>
                 <scope>provided</scope>
             </dependency>
-            <dependency>
-                <groupId>jetbrains.buildServer.agent</groupId>
```
Comment (mavlyutov) [inline pom.xml]: hmm, I cant understand how can it built without that

Comment (aefimov) [inline pom.xml]: It's declared in openstack-agent/pom.xml

File: pom.xml (hunk 2 above)
Comment (aefimov) [inline pom.xml]: Look deps here http://repository.jetbrains.com/all/org/jetbrains/teamcity/common-api/9.0/common-api-9.0.pom

**Pull request merged**